src/components/Event/Event.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { Text } from '../Text';
+import { EVENT_DATA_HOOKS, EVENT_DATA_KEYS } from './dataHooks';
+import styles from './Event.st.css';
+
+export interface EventProps {
```

Comment:
```suggestion
import styles from './Event.st.css';
import { TPAComponentProps } from '../../types';

export interface EventProps extends TPAComponentProps {
```
You should add this.
Sorry I thought it was part of the generator